build.sbt

```diff
@@ -219,10 +240,17 @@ val commonSettings = Def
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint:unchecked"),
     Compile / doc / javacOptions := Seq("-source", "1.8"),
     excludeDependencies ++= Seq(
-      "org.apache.beam" % "beam-sdks-java-io-kafka"
+      "org.apache.beam" % "beam-sdks-java-io-kafka",
+      "org.slf4j" % "slf4j-log4j12",
+      "org.slf4j" % "slf4j-reload4j"
```

Comment:
We should never ship logger impl with sio. Ans for tese, prefer the `slf4j-simple` implementation